doc/quickstart/install.rst

```diff
@@ -16,6 +16,37 @@ Once you have installed conda, you can install ESMValCore by running:
     conda install -c esmvalgroup -c conda-forge esmvalcore
 
 
+Using it through docker
+-----------------------
+
+ESMValCore is also provided thorugh DockerHub in the form of docker containers.
+See https://docs.docker.com/ for more info about docker containers and how to
+run the
+
+You can get the latest release with
+
+.. code-block:: bash
+
+   docker pull esmvalgroup/esmvalcore:stable
+
+If yoy want to use the current master branch, use
+
+.. code-block:: bash
+
+   docker pull esmvalgroup/esmvalcore:latest
+
+To run a container using those images, use:
+
+.. code-block:: bash
+
+   docker run esmvalgroup/esmvalcore:stable esmvaltool -v
+
```

Comment:
Could you also add instructions how to run with singularity? I think that many of our users will not be able to use docker, because they're on a cluster without root permissions.

Reply:
We have experienced problems trying to run with singularity in our cluster. I don't feel confident enough in the procedure to document it right now... but I just installed vagrant in my laptop to try to use the singularity image in https://app.vagrantup.com/sylabs to run it. Give me a couple of days

Reply:
Conversion is failing. I am trying to port the build recipe to singularity format

Reply:
Conversion is failing for the default conda image, but not for the previous version. I pinned the version to 4.7.12.  I am waiting for dockerhub to build the image to test it with singularity

Reply:
Ok. I finally manage to convert the image to singularity... by not using the vagrant image they provide and  use the conda-forge package I just discovered. I will add the documentation on how to use it

Reply:
Singularity works just fine for me, I installed it using [these instructions](https://sylabs.io/guides/3.0/user-guide/quick_start.html#quick-installation-steps) and then ran
```bash
singularity run -B $HOME:$HOME docker://esmvalgroup/esmvalcore:update_docker -c $HOME/.esmvaltool/config-user.yml $HOME/src/esmvalgroup/recipe_standalone.yml
```

Reply:
It will automatically convert and cache the image it downloads from dockerhub if you use the command above

Reply:
Yes, I also manage to convert the container to the singularity format and run it, both in my machine and our main supercomputer.

Reply:
> I am waiting for dockerhub to build the image to test it with singularity

You can also do this locally:
```
docker build -t test:latest . -f docker/Dockerfile
singularity build test.sif docker-daemon://test:latest
```

Anyway, now that it works, do you think we could add this to the documentation?

Reply:
Yes, I will do it. Yesterday was a bit chaotic here, that's why is no0t done yet